### ISSUE TITLE
I.L FE: Update <va-button-pair> component to latest version

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15330]
## Description 
The DST updated the `<va-button-pair>` component to resolve the accessibility issue regarding the incorrect tab order on small screens. We now need to update our component within the app on each screen to make sure it is using the latest version of the component in order to take advantage of this update.

## Acceptance Criteria
- [ ] Update `<va-button-pair>` on each screen/question to latest version
- [ ] Desktop tab order continues to follow DOM order
- [ ] Mobile tab order now follows correct DOM order
- [ ] Requires design review
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [x] Public Websites
- [ ] Facilities
- [ ] User support
- [ ] Accelerated Publishing

**Comment by wesrowe** [comment]
@randimays, no rush but please add your engineering context and pre-refine when you get a chance.

**Comment by laflannery** [comment]
This has been updated automagically, closing

**Comment by jilladams** [comment]
Related discussion: https://dsva.slack.com/archives/C52CL1PKQ/p1695226154712819
